### PR TITLE
cinnamon.xapps: 1.8.9 -> 2.0.6

### LINKED ISSUE #109475: cinnamon.xapps: upgrade to 1.9.0 or newer
###### Checklist

- [x] Checked the [nixpkgs master branch](https://github.com/NixOS/nixpkgs)

- [x] Checked the [nixpkgs pull requests](https://github.com/NixOS/nixpkgs/pulls)

###### Project name
`nix search` name: xapps

current version: 1.8.9
desired version: 1.9.0 or newer

###### Notify maintainers

maintainers: @mkg20001 

###### Note for maintainers

Please tag this issue in your PR.

### PR DESCRIPTION
**Pull request by tu-maurice**
###### Motivation for this change

Closes #109475

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/desktops/cinnamon/xapps/default.nix [hunk 1]
```diff
@@ -22,22 +22,15 @@
 
 stdenv.mkDerivation rec {
   pname = "xapps";
-  version = "1.8.9";
+  version = "2.0.6";
 
   outputs = [ "out" "dev" ];
 
-  patches = [
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/desktops/cinnamon/xapps/default.nix:30]: Please remove fetchpatch from inputs.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 110890` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
12 packages built:
</summary>
<ul>
<li>blueberry</li>
<li>cinnamon.cinnamon-common</li>
<li>cinnamon.cinnamon-gsettings-overrides</li>
<li>cinnamon.cinnamon-screensaver</li>
<li>cinnamon.cinnamon-session</li>
<li>cinnamon.cjs</li>
<li>cinnamon.nemo</li>
<li>cinnamon.warpinator</li>
<li>cinnamon.xapps</li>
<li>python37Packages.xapp</li>
<li>python38Packages.xapp</li>
<li>python39Packages.xapp</li>
</ul>
</details>

The following issues got detected with the above build packages.
Please fix at least the ones listed with your changed packages:

python37Packages.xapp:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python38Packages.xapp:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>
python39Packages.xapp:
<details>

Zero tests run by pytest got detected: 'Ran 0 tests in 0.000s'
You have the following options to fix this:
  - Tell pytest(CheckHook) where to find the tests included in the package
  - Check if the GitHub Repo contains tests but they are not shipped with Pypi. If so please switch to `fetchFromGitHub`.
  - If the Packages does not contain any tests add `'doCheck = false;'` and a `pythonImportsCheck`.

</details>

**Comment by tu-maurice** [comment]
Shall I solve the warnings for python-xapp in the same pull request or do a separate one and upgrade it as well?

**Comment by SuperSandro2000** [comment]
> Shall I solve the warnings for python-xapp in the same pull request or do a separate one and upgrade it as well?

I think you should do a separate PR here.

**Pull request merged**